plugins/module_utils/k8sdynamicclient.py

```diff
@@ -19,7 +19,7 @@
 
 from kubernetes.dynamic import DynamicClient
 
-from ansible_collections.kubernetes.core.plugins.module_utils.apply import apply
+from ansible_collections.kubernetes.core.plugins.module_utils.apply import k8s_apply
```

Comment:
``import apply as k8s_apply`` why not ?

Reply:
This is broken as well. The only way I can get this to work is to just rename the function.